doc/doxygen/src/roadmap.md

```diff
@@ -8,95 +8,75 @@ The text and items below are tentative, up for discussion, to be updated by regu
 
-1. remodeling  of the `periph/i2c.h` interface and subsequent adaption/rewrite of all existing implementations (-> https://github.com/RIOT-OS/RIOT/issues/6577)
-2. cleanup and unification of low-level timer interfaces (`timer`, `rtt`, `rtc`)
-3. introduction of `spi_slave` interface
-4. introduction of `i2c_slave` interface
+- cleanup and unification of low-level timer interfaces (`timer`, `rtt`, `rtc`)
+- introduction of `spi_slave` interface
+- introduction of `i2c_slave` interface
 
 
 
 # Software Updates
 (contact/steering: [Emmanuel](https://github.com/emmanuelsearch))
 
-1. ~~Basic bootloader, flashing and booting one of multiple firmware slots~~ done, see [riotboot](https://github.com/RIOT-OS/RIOT/tree/master/bootloaders/riotboot)
-2. ~~Secure firmware update over CoAP compliant with SUIT draft spec [draft-ietf-suit-manifest-03](https://tools.ietf.org/html/draft-ietf-suit-manifest-03)~~ done, see [SUIT update example](https://github.com/RIOT-OS/RIOT/tree/master/examples/suit_update)
-3. Update SUIT spec support to comply with [draft-ietf-suit-manifest-04](https://tools.ietf.org/html/draft-ietf-suit-manifest-04)
-4. Modularize to provide toolbox supporting other image storing (e.g. support external flash), transport (other than CoAP), crypto (e.g. secure element).
-5. riotboot support for architectures other than Cortex-M
+- Modularize to provide toolbox supporting other image storing (e.g. support external flash), transport (other than CoAP), crypto (e.g. secure element).
+- riotboot support for architectures other than Cortex-M
 
 
 
 # Documentation
 (contact/steering: [Emmanuel](https://github.com/emmanuelsearch))
 
-1. ~~publish roadmap~~ done
-2. ~~Introduce RIOT Developer Memos~~ done, see [RDM0](https://github.com/RIOT-OS/RIOT/tree/master/doc/memos)
-3. Write and publish more RDMs
-4. revamp RIOT website (see [Web Revamp Task Force](https://github.com/RIOT-OS/RIOT/wiki/Website-Revamp-Task-Force))
+- Write and publish more RDMs
 
 
 
 # Low-level Hardware Support
 (contact/steering: [Alex](https://github.com/aabadie))
 
-1. Improved MIPS support
-2. radio support for TI SensorTag
-3. radio support for Silab Thunderboard
-4. ESP32 support
+- radio support for TI SensorTag
+- radio support for Silab Thunderboard
 
 
 
 # Testing
 (contact/steering: [Kaspar](https://github.com/kaspar030))
 
-1. ~~automated unit tests with hardware in the loop (SAMR21 plugged on CI server?)~~ (done, Murdock and Philipp)
-2. automated network functionality tests (e.g. RPL + UDP/PING tests through border router, multi-hop) in IoTLAB dev sites?
-3. ~~leverage PiFleet more?~~ done, 1. uses PiFleet
-4. On-board CI testing in IoT-LAB (as it will provide soon the possibility to add custom nodes)
+- automated network functionality tests (e.g. RPL + UDP/PING tests through border router, multi-hop) in IoTLAB dev sites?
+- On-board CI testing in IoT-LAB (as it will provide soon the possibility to add custom nodes)
```

Comment:
> And yes, the release test should cover this.

(from https://github.com/RIOT-OS/RIOT/pull/20126/files#r1411960390)

In consequence, these can be removed ;-).



In consequence

Reply:
Done